### PR TITLE
chore: update changelog for 2.6.2 bugfix release

### PR DESCRIPTION
**Pull request by olevski**
Updating the changelog for the release in hotfix/2.6.1 branch

### CODE REVIEW

**Review by Panaetius — DISMISSED** [review]

File: CHANGES.rst [hunk 1]
```diff
@@ -48,6 +48,44 @@ Features
...
+`2.6.2 <https://github.com/SwissDataScienceCenter/renku-python/compare/v2.6.1...v2.6.2>`__ (2023-08-23)
+-------------------------------------------------------------------------------------------------------
+
+Bug Fixes
+~~~~~~~~~
+
+-  **cli:** correct bad plan ID format
+   (`#3594 <https://github.com/SwissDataScienceCenter/renku-python/issues/3594>`__)
+   (`c418c17 <https://github.com/SwissDataScienceCenter/renku-python/commit/c418c178d03a5caac126d14cc089064ee13f2747>`__)
+-  **cli:** do not freeze/unfreeze plan view model
```
Comment (mohammad-alisafaee) [inline CHANGES.rst:60]: This shouldn't be here since it's not a user facing issue.

File: CHANGES.rst [hunk 2]
```diff
@@ -48,6 +48,44 @@ Features
...
+   (`#3599 <https://github.com/SwissDataScienceCenter/renku-python/issues/3599>`__)
+   (`dab2688 <https://github.com/SwissDataScienceCenter/renku-python/commit/dab26884170b0277767bb0ad5aa1d7e3472243bf>`__)
+-  **cli:** fix overwriting dataset tags
+   (`#3497 <https://github.com/SwissDataScienceCenter/renku-python/issues/3497>`__)
+   (`ef6d214 <https://github.com/SwissDataScienceCenter/renku-python/commit/ef6d214e165ba877b5acc26427d5663366aaaa29>`__)
+-  **cli:** parse security_opt as list not tuple
```
Comment (mohammad-alisafaee) [inline CHANGES.rst:66]: Same as above

File: CHANGES.rst [hunk 3]
```diff
@@ -48,6 +48,44 @@ Features
...
+   (`#3587 <https://github.com/SwissDataScienceCenter/renku-python/issues/3587>`__)
+   (`d74d35f <https://github.com/SwissDataScienceCenter/renku-python/commit/d74d35ff3428118945a1df1b2f52eebcfcf31132>`__)
+-  **core:** properly formatted dataset image IDs
+   (`#3596 <https://github.com/SwissDataScienceCenter/renku-python/issues/3596>`__)
+   (`f624b2b <https://github.com/SwissDataScienceCenter/renku-python/commit/f624b2bf261d97b07c88243f674f544613753e28>`__)
+-  **service:** remove doctor check from cache.migrations_check
```
Comment (mohammad-alisafaee) [inline CHANGES.rst:72]: I'd rephrase this as `speed up cache.migration_check endpoint`.

File: CHANGES.rst [hunk 4]
```diff
@@ -48,6 +48,44 @@ Features
...
+   (`#3597 <https://github.com/SwissDataScienceCenter/renku-python/issues/3597>`__)
+   (`20b5589 <https://github.com/SwissDataScienceCenter/renku-python/commit/20b5589ea2639b4ff017fc390a9b685842c9685d>`__)
+-  **service:** return project version as int
```
Comment (mohammad-alisafaee) [inline CHANGES.rst:75]: Either remove this or `serialize project versions as integer`!?

**Review by mohammad-alisafaee — COMMENTED** [review]
Thanks Tasko! I've made some suggestions regarding the thing that should go in the release note. Feel free to disagree.

File: CHANGES.rst (hunk 2 above)
Comment (olevski) [reply CHANGES.rst]: This bug was reported by a user. It prevents people from launching sessions with the CLI. So I kept it but reworded it.

File: CHANGES.rst (hunk 3 above)
Comment (olevski) [reply CHANGES.rst]: Done.

**Review by mohammad-alisafaee — APPROVED** [review]
